apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js

```diff
@@ -61,18 +61,18 @@ function WhatsNewMenuItem() {
 function getWhatsNewPages() {
 	return [
 		{
-			imgSrc: blockPatternsImage,
-			heading: __( 'New block patterns', 'full-site-editing' ),
+			imgSrc: universalNavImage,
+			heading: __( 'Easier navigation', 'full-site-editing' ),
 			description: createInterpolateElement(
 				/* translators: the embed is a link */
 				__(
-					'<p>Choose from hundreds of pre-made patterns for buttons, headers, galleries, and more available via the + button at the top of all pages.</p><p><Link>Learn more</Link></p>',
+					'<p>This month all WordPress.com accounts will receive sidebar and menu updates, making it easier to manage your site from the left sidebar.</p><p><Link>Learn more</Link></p>',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( '<p>This month all WordPress.com accounts will receive sidebar and menu updates, making it easier than ever to manage your site from the left sidebar.</p><p><Link>Learn more</Link></p>' )` *ES Score: 9*